### ISSUE TITLE
Hard to navigate gokibitz on smartphone. 

### ISSUE THREAD

**Issue by levelonedev** [neagle/gokibitz#53]
It is hard to find new comments since you can't see them below. Right now I continue to the front page of the kifu every time to click on the next comment listed there. The new button/functionality for browsing to the next comment should help a lot.
![image](https://cloud.githubusercontent.com/assets/9813893/6757912/49a27706-cefa-11e4-9716-23c568d0da01.jpg)

**Comment by neagle** [comment]
Yeah, this has been on my mind, too. I think we need something more than just the comment browsing... I've been thinking about some kind of small indicator that's level with the board that shows the number of comments and jump-links to the comment section.

**Comment by lukas1** [comment]
What would also help on mobile devices is, that when you tap on the right side of the board, you'd go one move forward, when you tap on the left side of the board, you'd go one move backwards. Right now it's difficult to navigate move by move and a mis-tap on skip next 10 moves happens to me frequently

**Comment by neagle** [comment]
I could think about something like that. One note: you can also swipe left or right to navigate.

--
Nate Eagle
